R/pkg/R/functions.R

```diff
@@ -2876,7 +2876,8 @@ setMethod("randn", signature(seed = "numeric"),
 
 #' regexp_extract
 #'
-#' Extract a specific(idx) group identified by a java regex, from the specified string column.
+#' Extract a specific \code{idx} group identified by a Java regex, from the specified string column.
```

Comment:
I noticed that - I thought `@param idx a group index` isn't super descriptive on what it is